<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetManual</class>
 <widget class="QWidget" name="SetManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="humidityButton">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>80</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Add 0ml of Water</string>
   </property>
  </widget>
  <widget class="QPushButton" name="windowButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Open Windows</string>
   </property>
  </widget>
  <widget class="QSlider" name="humiditySlider">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>0</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="heaterButton">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>80</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Start Heater</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="overrideWindowBox">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
width: 20px;
height: 20px;
}</string>
   </property>
   <property name="text">
    <string>Override auto.</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="overrideHeaterBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
width: 20px;
height: 20px;
}</string>
   </property>
   <property name="text">
    <string>Override auto.</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="deviceComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
